src/main/java/view/DashboardView.java

```diff
@@ -81,93 +96,201 @@ public DashboardView(DashboardViewModel dashboardViewModel,
-                new ActionListener() {
-                    public void actionPerformed(ActionEvent evt) {
-                        if (evt.getSource().equals(buy)) {
-                            buyViewModel.firePropertyChanged();
-                            viewManagerModel.setActiveView(buyViewModel.getViewName());
-                            viewManagerModel.firePropertyChanged();
-                        }
-                    }
+                evt -> {
+                    buyViewModel.firePropertyChanged();
+                    viewManagerModel.setActiveView(buyViewModel.getViewName());
+                    viewManagerModel.firePropertyChanged();
                 }
         );
 
         sell.addActionListener(
-                new ActionListener() {
-                    public void actionPerformed(ActionEvent evt) {
-                        if (evt.getSource().equals(sell)) {
-                            sellViewModel.firePropertyChanged();
-                            viewManagerModel.setActiveView(sellViewModel.getViewName());
-                            viewManagerModel.firePropertyChanged();
-                        }
-                    }
+                evt -> {
+                    sellViewModel.firePropertyChanged();
+                    viewManagerModel.setActiveView(sellViewModel.getViewName());
+                    viewManagerModel.firePropertyChanged();
                 }
         );
 
         news.addActionListener(
-                new ActionListener() {
-                    public void actionPerformed(ActionEvent evt) {
-                        if (evt.getSource().equals(news)) {
-                            getNewsViewModel.firePropertyChanged();
-                            viewManagerModel.setActiveView(getNewsViewModel.getViewName());
-                            viewManagerModel.firePropertyChanged();
-
-                            // TODO: Do we then have to call GetNewsController.execute() to obtain
-                            //  the needed information?
-                        }
-                    }
+                evt -> {
+                    getNewsViewModel.firePropertyChanged();
+                    viewManagerModel.setActiveView(getNewsViewModel.getViewName());
+                    viewManagerModel.firePropertyChanged();
                 }
         );
 
         reset.addActionListener(
-                new ActionListener() {
-                    public void actionPerformed(ActionEvent evt) {
-                        if (evt.getSource().equals(reset)) {
-                            resetBalanceController.execute();
-                            dashboardViewModel.firePropertyChanged();
-
-                            // TODO: add reset balance pop-up
+                evt -> {
+                    resetBalanceController.execute();
+                    dashboardViewModel.firePropertyChanged();
 
-                        }
-                    }
+                    // TODO: add reset balance pop-up
```

Comment:
I dont think we are doing the pop-up. add it as an issue

Reply:
So you're saying I should remove the comment and then create an issue for it... ?